### ISSUE TITLE
Error when extended iterable unpacking used in method

### ISSUE THREAD

**Issue by joouha** [bwhmather/ssort#28]
Running `ssort` on anything where [extended iterable unpacking](https://www.python.org/dev/peps/pep-3132/) is used in a class method leads to an error.

Sorting the following:

```python
class MyClass:
    def method(self):
        a, *b = 1, 2, 3
```

leads to the following exception:

```
Traceback (most recent call last):
  File "/home/josiah/projects/ssort/src/ssort/_main.py", line 64, in main
    updated = ssort(original, filename=str(path))
  File "/home/josiah/projects/ssort/src/ssort/_ssort.py", line 333, in ssort
    output = "\n".join(
  File "/home/josiah/projects/ssort/src/ssort/_ssort.py", line 334, in <genexpr>
    statement_text_sorted(statement) for statement in sorted_statements
  File "/home/josiah/projects/ssort/src/ssort/_ssort.py", line 318, in statement_text_sorted
    return _statement_text_sorted_class(statement)
  File "/home/josiah/projects/ssort/src/ssort/_ssort.py", line 297, in _statement_text_sorted_class
    runtime_graph = class_statements_runtime_graph(sorted_statements)
  File "/home/josiah/projects/ssort/src/ssort/_dependencies.py", line 117, in class_statements_runtime_graph
    for name in statement_method_requirements(statement):
  File "/home/josiah/projects/ssort/src/ssort/_method_requirements.py", line 740, in _get_method_requirements_for_function_def
    yield from _get_attribute_accesses(statement, self_arg)
  File "/home/josiah/projects/ssort/src/ssort/_method_requirements.py", line 95, in _get_attribute_accesses_for_assign
    yield from _get_attribute_accesses(target, variable)
  File "/home/josiah/projects/ssort/src/ssort/_method_requirements.py", line 666, in _get_attribute_accesses_for_tuple
    yield from _get_attribute_accesses(element, variable)
  File "/home/josiah/projects/ssort/src/ssort/_method_requirements.py", line 629, in _get_attribute_accesses_for_starred
    assert isinstance(node.ctx, ast.Load)
AssertionError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/josiah/projects/ssort/src/ssort/__main__.py", line 3, in <module>
    main()
  File "/home/josiah/projects/ssort/src/ssort/_main.py", line 85, in main
    raise Exception(f"ERROR while sorting {path}\n") from e
Exception: ERROR while sorting test.py
```

**Comment by bwhmather** [comment]
Not just that.  Looks like anything complex in an assignment target was being parsed as an rvalue.  I think I've managed to fix the slightly more general problem in #30, but I'm not sure that it won't recur with pattern matching in 3.10.  Please bump to version 0.10.2 for the fix.  Thank you very much for reporting

**Comment by joouha** [comment]
Wonderful, thanks!
